### PR TITLE
[CH-78] Add microbenchmark for ch shuffle operator

### PR DESCRIPTION
**Pull request by zzcclp**
## What changes were proposed in this pull request?

Add microbenchmark for ch shuffle operator

## How was this patch tested?

no tests required

### CODE REVIEW

**Comment by zzcclp** [comment]
Delete some unused code.

**Review by zhouyuan — APPROVED** [review]
👍

File: backends-clickhouse/src/test/scala/org/apache/spark/sql/execution/benchmarks/CHAggAndShuffleBenchmark.scala [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+                                 |     $groupByKeys
+                                 |     ${if (aggFunctions.nonEmpty) "," + aggFunctions else ""}
+                                 | FROM
+                                 |     $fileFormat.`$parquetDir`
+                                 | $filters
+                                 | GROUP BY
+                                 |     $groupByKeys) as a
+                                 |""".stripMargin)
+
+    val executedPlan = allStages.queryExecution.executedPlan
```
Comment (baibaichen) [inline backends-clickhouse/src/test/scala/org/apache/spark/sql/execution/benchmarks/CHAggAndShuffleBenchmark.scala:163]: It looks good, but it's better to add more comments to describe the stages of this SQL

Comment (zzcclp) [reply backends-clickhouse/src/test/scala/org/apache/spark/sql/execution/benchmarks/CHAggAndShuffleBenchmark.scala:177]: Done.

Comment (baibaichen) [reply backends-clickhouse/src/test/scala/org/apache/spark/sql/execution/benchmarks/CHAggAndShuffleBenchmark.scala:177]: Thanks!

**Pull request merged**